go/tools/go_bin_runner/main.go

```diff
@@ -48,12 +51,16 @@ func main() {
 	}
```

Comment:
Random stylistic thing: 

I would be good to separate out the logic into a `run()` function that returns an error, so that we only need to call `log.Fatal` once. Then we can test this stuff in the future:

```
func main() {
  if err := run(os.Args, os.Stderr); err != nil {
    log.Fatal(err)
  }
}

func run(args []string, stderr io.Writer) error {
  .. everything else
}
```